src/main/resources/application.properties

```diff
@@ -11,8 +11,8 @@ security.user.password=81d56928-891a-41b2-8dd2-132593e6d057
 # database url and language
 # H2
 spring.datasource.platform=h2
-spring.datasource.url=jdbc:h2:mem:AZ;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE
-#spring.datasource.url=jdbc:h2:file:./h2/app.db;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE
+#spring.datasource.url=jdbc:h2:mem:AZ;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE
```

Comment:
What settings are we pushing? Should stay the same for convenience of development. Even though not to hard to cut and past url into admin console login.